ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/DLModel.java

```diff
@@ -253,6 +253,7 @@ protected void loadModel(
                 ClassLoader contextClassLoader = Thread.currentThread().getContextClassLoader();
                 try {
                     System.setProperty("PYTORCH_PRECXX11", "true");
+                    System.setProperty("PYTORCH_VERSION", "1.13.1");
```

Comment:
Why not `2.2.2`?

Reply:
Yeah ideally it would pick up 2.2.2 but djl team suggested using 1.13.1 which is more stable